src/web/components/PortalHeader/PortalHeader.spec.tsx

```diff
@@ -17,4 +44,67 @@ describe('Portal Header tests', () => {
     const button = screen.getByRole('button');
     expect(button).toHaveTextContent('Not logged in');
   });
+
+  describe('Audit Trail in Portal Header tests based on role', () => {
+    const user = userEvent.setup();
+    let participantContextValue: ParticipantWithSetter;
+    let userContextValue: UserContextWithSetter;
+    let mockParticipant: ParticipantDTO;
+    let mockUser: UserWithParticipantRoles;
+
+    beforeAll(() => {
+      mockParticipant = createMockParticipant();
+      participantContextValue = createParticipantContextValue(mockParticipant);
+      mockUser = createMockUser([mockParticipant] as Participant[]);
+      userContextValue = createUserContextValue(mockUser);
+    });
+
+    it('should show Audit Trail when user has uid2support role', async () => {
```

Comment:
nit: the existing tests use `test` while the new tests use `it`. They do the same thing but we should probably just use the same one everywhere (at least stick to one within the same file)

Reply:
Fixed - i like `it` better